formulate-a-premise/readme.md

```diff
@@ -0,0 +1,15 @@
+>Goal of this guide: teach you how to formulate a good premise for your story
+
+# How to formulate a premise
+
+## What is a premise and how do I devise one effectively?
+* A [storyline](https://en.wikipedia.org/wiki/Storyline)
+* A [theme](https://en.wikipedia.org/wiki/Theme_(narrative))
```

Comment:
These are the answers to the question 
> What is a premise?

I didn't get it after I read 
> how do I devise one effectively?

Maybe place the second question above its rightful questions?